src/toplevel_generators.jl

```diff
@@ -83,8 +83,8 @@ function generate_toplevel_definitions(src_name, __source__)
         """
         function is_available end
 
-        PATH = ""
-        LIBPATH = ""
+        const PATH = Ref{String}("")
```

Comment:
I wasn't sure if these were supposed to be initialized or not but if they are not I run into https://github.com/JuliaPackaging/JLLWrappers.jl/issues/11.